libfuse/dir.go

```diff
@@ -260,6 +280,7 @@ func (f *Folder) batchChangesInvalidate(ctx context.Context,
 // TlfHandleChange is called when the name of a folder changes.
 func (f *Folder) TlfHandleChange(ctx context.Context,
 	newHandle *libkbfs.TlfHandle) {
+	f.fs.log.CDebugf(ctx, "TlfHandleChange called")
```

Comment:
Done. (although for debugging whether TlfHandleChange is called at all was sufficient).